main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player, values in players.items():
+        nickname, email, bio, race = (
+            player,
+            values["email"],
+            values["bio"],
+            values["race"],
+        )
+
+        race_name, race_description = race["name"], race["description"]
+
+        race_obj, _ = Race.objects.get_or_create(
+            name=race_name, description=race_description
+        )
+
+        skills = race["skills"]
+        for skill in skills:
+            name = skill["name"]
+            bonus = skill["bonus"]
+
+            Skill.objects.get_or_create(name=name, bonus=bonus, race=race_obj)
+
+        guild = values.get("guild")
```

Comment:
read what get method will return if key does not exist, mb we can refactor this place